cfgov/unprocessed/js/modules/util/fnBind.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+
+/**
+* Function.Bind polyfill.
+*
+* @access private
+* @function fnBind
+* @param {function} fn - a function you want to change `this` reference to
+* @param {object} context - the `this` you want to call the function with
+* @returns {function} The wrapped version of the supplied function
+*/
+
+function fnBind(fn, context) {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1866/commits/92ac43e13793816b3f1c1466ccf5f94e0b255dff